Cargo.toml

```diff
@@ -66,11 +67,12 @@ dora-coordinator = { version = "0.3.4", path = "binaries/coordinator" }
 dora-ros2-bridge = { path = "libraries/extensions/ros2-bridge" }
 dora-ros2-bridge-msg-gen = { path = "libraries/extensions/ros2-bridge/msg-gen" }
 dora-ros2-bridge-python = { path = "libraries/extensions/ros2-bridge/python" }
+python_special_method_derive = { path = "python_special_method_derive" }
 arrow = { version = "52" }
 arrow-schema = { version = "52" }
 arrow-data = { version = "52" }
 arrow-array = { version = "52" }
-pyo3 = "0.21"
+pyo3 = { version = "0.21", features = ["eyre", "abi3-py37", "multiple-pymethods"] }
```

Comment:
So basically, people wanting to use this crate will need to have the pyo3/multiple-pymethods feature flag on right?

I think it's something worth documenting somewhere.

Reply:
Yes. I'll document this clearly in the new crate.